ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.1.yaml

```diff
@@ -0,0 +1,39 @@
+binary_build_commands: |
+  cat > /bin/openstack-tests <<EOF
+  #!/usr/bin/env bash
+  echo 'Not implemented'
+  EOF
+  chmod +x /bin/openstack-tests
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
Please check the golang version each branch is pinned to.

Reply:
I think 1.16 is fine for all branches. Do you think we should change it?

Reply:
Not sure, I was under the impression each ocp release was capped to a golang version but I might be mistaken.